src/app/containers/Headings/index.test.jsx

```diff
@@ -74,6 +78,8 @@ const template = (title, text, type) => {
 
 describe('Headings', () => {
   describe('with no data', () => {
+    suppressPropWarnings(['type', 'undefined']);
+    suppressPropWarnings(['blocks', 'HeadingsContainer']);
```

Comment:
Would change ```'HeadingsContainer'``` to ```supplied``` in order to make it clearer what kind of warning is being suppressed if for whatever reason this test needs to be looked at again by someone else in the future.